### ISSUE TITLE
Camera app face ID not working after jb xs max 12.4

### ISSUE THREAD

**Issue by Heftydose** [pwn20wndstuff/Undecimus#1552]
**Describe the bug**
Cannot open camera app and face ID :(  
 
I hope they will fix this:( miss my jailbreak tweaks🙏🙏🙌🥺

**Comment by spalekg** [comment]
I can confirm. When JB is active camera doesn't work. After I have rebooted phone (JB not active) camera works well.

iP XS max iOS 12.4

**Comment by Heftydose** [comment]
Yeah thats right. I even use ldrestart as temporary way to resolve the issue. Hopefully they will include and fix this issue on the next beta

**Comment by abigchopstick** [comment]
no temporary fix works for me now.

When I first JB my phone it worked, and then after install some tweaks it stopped working, my high suspect is AudioRecorder XS.

The 2nd time I installed the JB, GPS immediately stopped working, but camera worked until I installed some tweaks...

The 3rd time I tried...it stopped working right after I JB, so...I'm waiting for an update now.

BTW, I've tried all temp fixes like LDRESTART, take photos, remove AudioRecorder XS, unload tweaks and daemons while Jailbreaking, then iCleaner respring etc. Some made it work for a short while and stopped working again...

so...if you have a temp fix for you, you are lucky and just stick to it until it stop working for you. Good luck
